modules/phase_field/src/materials/OPInterfaceBarrierMaterial.C

```diff
@@ -0,0 +1,83 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+#include "OPInterfaceBarrierMaterial.h"
+
+template<>
+InputParameters validParams<OPInterfaceBarrierMaterial>()
+{
+  InputParameters params = validParams<Material>();
+  params.addParam<std::string>("function_name", "g", "actual name for g(eta_i)");
+  MooseEnum h_order("SIMPLE=0", "SIMPLE");
+  params.addParam<MooseEnum>("g_order", h_order, "Polynomial order of the switching function h(eta)");
```

Comment:
g_order or h_order. This should be consistent, or - preferably - removed, since there is only one option now.